src/components/EditImageModal/EditImageModal.scss

```diff
@@ -1,7 +1,7 @@
 @import 'edx-bootstrap';
 
 .modal-body {
-  height: 400px;
+  height: 75%;
```

Comment:
I wonder if we could set a max-width instead here. I think the modal will get really smooshed if the window height is scaled way down.